packages/contentstack-bulk-publish/src/util/bulk-publish-utils.js

```diff
@@ -0,0 +1,39 @@
+const { regions } = require('@contentstack/cli-config/lib/utils/region-handler');
```

Comment:
can you name this file better, since the folder tells it that is a group of utilities, give name specific to its usage for better understanding such as "generate-bulk-publish-url.js"